### PR TITLE
Remove Chris Bane's Bitmap Cache library

### PR DESCRIPTION
**Pull request by HenriqueRocha**
To address issue #190 I've removed Chris Bane's Bitmap Cache library from the project and now only Picasso is used for loading images.
When I have more time I'll also change from Volley to Retrofit but I'll work on other things first.

### CODE REVIEW

File: app/src/main/assets/third_party.html [hunk 1]
```diff
@@ -140,21 +140,6 @@
...
-You may obtain a copy of the License at<br/><br/>
-
-http://www.apache.org/licenses/LICENSE-2.0<br/><br/>
-
-Unless required by applicable law or agreed to in writing, software<br/>
-distributed under the License is distributed on an "AS IS" BASIS,<br/>
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.<br/>
-See the License for the specific language governing permissions and<br/>
-limitations under the License.<br/><br/>
-
```
Comment (stefanhoth) [inline app/src/main/assets/third_party.html]: :+1: that you thought of that

**Comment by stefanhoth** [comment]
Good effort but I'm a bit puzzled because you didn't change any code that used Volley for image loading. Was that part already replaced before?

**Comment by HenriqueRocha** [comment]
@stefanhoth Yes, I was a bit puzzled myself when I removed the code from GdgVolley class.
It was initializing the bitmap cache but not actually using it for anything at all.

**Comment by stefanhoth** [comment]
Weird. Thanks for removing dead code then. :+1:

**Pull request merged**